sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2581,6 +2581,59 @@ abstract class SQLQuerySuiteBase extends QueryTest with SQLTestUtils with TestHi
       }
     }
   }
+
+  test("xxx") {
```

Comment:
Looks like we should have a proper test name btw

Reply:
> Looks like we should have a proper test name btw

Hmmmm, updated ==